src/main/java/org/takes/rs/RsBuffered.java

```diff
@@ -47,17 +45,10 @@
      */
     public RsBuffered(final Response res) {
         super(
-            new Response() {
-                @Override
-                public Iterable<String> head() throws IOException {
-                    return res.head();
-                }
-
-                @Override
-                public InputStream body() throws IOException {
-                    return new BufferedInputStream(res.body());
-                }
-            }
+            new ResponseOf(
+                res::head,
+                () -> new BufferedInputStream(res.body())
```

Comment:
@fanifieiev same above.

Reply:
@fabriciofx I cannot use secondary constructor here.
![image](https://user-images.githubusercontent.com/3167940/69073037-9dac2500-09fa-11ea-9d66-dd380b525573.png)

Reply:
@fanifieiev as I said [here](https://github.com/yegor256/takes/pull/995#discussion_r347074093): *Of couse you can't use it in all places (e.g. beacause it can throws an Exception). So, in these places you can use the primary constructor...*